test/dynamicBaseFee.js

```diff
@@ -381,6 +381,10 @@ describe('Have to set correct value optional fields named gasPrice, maxFeePerGas
         // Generate many txs to increase baseFee
         generateTxsBomb(await fillKlay(100))
 
+        try {
+            // If account is already existed in node, return error.
```

Comment:
The comment seems little bit weird. so, if await throws an error, it returns an error?

Reply:
Ah i mean Node returns an error if there is already existed account. I will modify the comment